config/locales/assessor_interface.en.yml

```diff
@@ -153,55 +153,55 @@ en:
 
       failure_reasons:
         as_statement:
-          additional_degree_certificate_illegible: An additional degree certificate (or translation) is illegible or in a format that we cannot accept.
```

Comment:
These are the failure reasons as shown to the assessor, but the ones we show to the teacher are here instead:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/22dfe3ee3efaf7fdea747c0803a8ef86270409b0/config/locales/teacher_interface.en.yml#L6

If we want them to be the same (those shown to the assessor and those shown to the teacher) then we should delete one of these sets of failure reasons to avoid duplication.